### PR TITLE
Update 'Environment Variables' section in README for the 'ECS_ENABLE_…

### PR DESCRIPTION
**Pull request by aaithal**
…TASK_IAM_ROLE' variable.

Also update 'ECS_AVAILABLE_LOGGING_DRIVERS' example with 'awslogs'.

r? @samuelkarp

### CODE REVIEW

**Comment by samuelkarp** [comment]
Can we add the [full list](https://github.com/aws/amazon-ecs-agent/blob/v1.11.0/agent/engine/dockerclient/logging_drivers.go)?  Might be nice to order them alphabetically too.

**Comment by aaithal** [comment]
done

File: README.md [hunk 1]
```diff
@@ -106,12 +106,13 @@ configure them as something other than the defaults.
...
-| `ECS_AVAILABLE_LOGGING_DRIVERS` | `["json-file","splunk","syslog"]` | Which logging drivers are available on the Container Instance. | `["json-file"]` |
+| `ECS_AVAILABLE_LOGGING_DRIVERS` | `["awslogs","fluentd","gelf","json-file","journald","splunk","syslog"]` | Which logging drivers are available on the Container Instance. | `["json-file"]` |
 | `ECS_DISABLE_PRIVILEGED` | `true` | Whether launching privileged containers is disabled on the Container Instance. | `false` |
 | `ECS_SELINUX_CAPABLE` | `true` | Whether SELinux is available on the Container Instance. | `false` |
 | `ECS_APPARMOR_CAPABLE` | `true` | Whether AppArmor is available on the Container Instance. | `false` |
 | `ECS_ENGINE_TASK_CLEANUP_WAIT_DURATION` | 10m | Time to wait to delete containers for a stopped task. If set to less than 1 minute, the value will be ignored.  | 3h |
 | `ECS_CONTAINER_STOP_TIMEOUT` | 10m | Time to wait for the container exit normally before being forcibly killed. | 30s |
+| `ECS_ENABLE_TASK_IAM_ROLE` | `false` | Whether to enable IAM Roles for Tasks on the Container Instance | `true` |
```
Comment (samuelkarp) [inline README.md]: Default value here is wrong; the default is `false`.

Comment (aaithal) [inline README.md]: ah, my bad. fixed now.

**Comment by samuelkarp** [comment]
:+1:

**Comment by aaithal** [comment]
merged

**Pull request closed without merging**